src/ArrayHelper.php

```diff
@@ -630,17 +630,29 @@ public static function multisort(array &$array, $key, $direction = SORT_ASC, $so
         if (empty($keys) || empty($array)) {
             return;
         }
+
```

Comment:
The method becomes too big.
Should be refactored to a separate class?

Reply:
@samdark ?

Reply:
How would you name such a class?

Reply:
Let's handle it separately: https://github.com/yiisoft/arrays/issues/29